src/StackExchange.Exceptional.AspNetCore/ExceptionalMiddleware.cs

```diff
@@ -49,7 +49,7 @@ public class ExceptionalMiddleware
         /// <summary>
         /// Executes the Exceptional-wrapped middleware.
         /// </summary>
-        public async Task Invoke(HttpContext context)
+        public async Task Invoke(HttpContext context)            
```

Comment:
Removed white spaces. Don´t know if I should remove the commit at all or only whitespaces. Please let me know :)